config.yml

```diff
@@ -0,0 +1,37 @@
+sonatypeWork: "/sonatype-work"
```

Comment:
With this, do we now need to maintain this to keep it in sync with insight-brain repo?

Reply:
> With this, do we now need to maintain this to keep it in sync with insight-brain repo?

Somewhat, it doesn't need to be an exact duplicate of `config.yml` in insight-brain, but if we update the docker image to use a new brain version, then we need to make sure the `config.yml` still makes sense. This is similar to the situation with chef (the only difference is chef was programmatically creating its `config.yml` but whatever it created was equally at risk of being outdated)

Reply:
> > With this, do we now need to maintain this to keep it in sync with insight-brain repo?
> 
> Somewhat, it doesn't need to be an exact duplicate of `config.yml` in insight-brain, but if we update the docker image to use a new brain version, then we need to make sure the `config.yml` still makes sense. This is similar to the situation with chef (the only difference is chef was programmatically creating its `config.yml` but whatever it created was equally at risk of being outdated)

(Just throwing out a wild idea, feel free to ignore). What if we we keep this in insight-brain just like other config.yml versions (say `config-docker.yml` or something) so that we can maintain it along with PRs if required. And if we package it in the root of the IQ distribution, we should be able to grab it after the extracting the downloaded distribution above.

Reply:
> (Just throwing out a wild idea, feel free to ignore). What if we we keep this in insight-brain just like other config.yml versions (say `config-docker.yml` or something) so that we can maintain it along with PRs if required. And if we package it in the root of the IQ distribution, we should be able to grab it after the extracting the downloaded distribution above.

I can see the advantage (easier for us to maintain) but there's also the disadvantage of some customers may not care about docker, it's unfortunate the files are different. Personally i can see it working either way. On the plus side we generally keep backwards compatibility with older `config.yml` files so the chance of a given `config.yml` file breaking here should be low